tests/unit/files/datasets/test_delete.py

```diff
@@ -0,0 +1,48 @@
+import re
+from unittest import TestCase, mock
+
+from zowe.zos_files_for_zowe_sdk import Files, exceptions, Datasets
+
+
+class TestDeleteClass(TestCase):
+    """File class unit tests."""
+
+    def setUp(self):
+        """Setup fixtures for File class."""
+        self.test_profile = {
+            "host": "mock-url.com",
+            "user": "Username",
+            "password": "Password",
+            "port": 443,
+            "rejectUnauthorized": True,
+        }
+
+    @mock.patch("requests.Session.send")
+    def test_delete(self, mock_send_request):
+        """Test list members sends request"""
+        mock_send_request.return_value = mock.Mock(headers={"Content-Type": "application/json"}, status_code=204)
+
+        Files(self.test_profile).delete_data_set(dataset_name="ds_name", member_name="member_name")
+        mock_send_request.assert_called_once()
+
+    @mock.patch("requests.Session.send")
+    def test_delete(self, mock_send_request):
```

Comment:
There are two methods both called `test_delete` in this file - one or both of them should be renamed.